spec/requests/issues_spec.rb

```diff
@@ -2,9 +2,48 @@
 
 RSpec.describe "Issues", type: :request do
   describe "GET /issues" do
-    it "works! (now write some real specs)" do
+    let(:user) {create(:user)}
+    let!(:issue) {create(:issue, user_id: user.id, principal: "Starbucks")}
+    let!(:issue2) {create(:issue, user_id: user.id)}
+    let!(:issue3) {create(:issue, user_id: user.id)}
+    it "returns all of the issues" do
       get issues_path
+
+      expect(response).to have_http_status(200)
+      response_body = JSON.parse(response.body)
+      expect(response_body.count).to eq 3
+      expect(response_body[0]["principal"]).to eq "Starbucks"
```

Comment:
When testing it is easier to check if one hash is equal to another rather than individually checking it's values.

Reply:
I think its easier to read for a new person coming in and looking at the spec though